kiwi_lint/field_label.py

```diff
@@ -0,0 +1,27 @@
+from pylint import checkers
+from pylint import interfaces
```

Comment:
Rename the module to `forms.py` b/c there are other checkers to add in this space. That will be more generic.